spanner/client.go

```diff
@@ -509,6 +511,14 @@ func ApplyAtLeastOnce() ApplyOption {
 	}
 }
 
+// Priority returns an ApplyOptions that sets the RPC priority to use for the
+// commit operation.
+func Priority(priority sppb.RequestOptions_Priority) ApplyOption {
```

Comment:
Maybe this is not true for spanner, but I don't think we directly expose the raw protos in other manual layers. Is there a Priority struct in this package that can be used and mapped to instead?

Reply:
Specifically for proto `enum` (option) values we do expose the proto values do avoid having to manually add and maintain a copy of the enum in the manual client. See for example these options: https://github.com/googleapis/google-cloud-go/blob/5fa2f7516d0c46a4c5fe8660e8138edc11afad17/spanner/transaction.go#L231-L234